packages/docs/src/components/header/Header.vue

```diff
@@ -127,48 +127,18 @@ export default class Header extends mixins(PropsMixin) {
       .header__nav {
         margin-left: 0.5rem;
       }
-
-      .header__links__button {
-        --va-button-content-px: 0.25rem;
-
-        margin-right: 0;
-        margin-left: 0.25rem;
-      }
-
-      &__links {
-        justify-content: flex-end !important;
-
-        @media screen and (max-width: 479px) {
-          justify-content: center !important;
-        }
-      }
-
-      &__prefences {
-        flex: 1 1 !important;
-        justify-content: space-between;
-
-        .language-dropdown {
-          display: flex;
-          justify-content: flex-end;
-        }
-
-        & .va-dropdown__anchor,
-        .color-dropdown {
-          display: flex;
-          justify-content: center;
-        }
-      }
     }
 
-    &__links {
+    &__prefences {
       display: flex;
+      align-items: center;
+      font-weight: $font-weight-bold;
```

Comment:
Do not scss variables. Use css variables or hardocde :)

Reply:
It was here before - will remove.